assets/js/modules/analytics/setup.js

```diff
@@ -1102,7 +1102,8 @@ class AnalyticsSetup extends Component {
 				}
 
 				{ !! existingTag &&
-					<p>{ sprintf( __( 'An existing analytics tag was found on your site with the id %s. If later on you decide to replace this tag, Site Kit can place the new tag for you. Make sure you remove the old tag first.', 'google-site-kit' ), existingTag ) }</p>
+					/* translators: %s: Analytics tag ID */
+					<p>{ sprintf( __( 'An existing analytics tag was found on your site with the ID %s. If later on you decide to replace this tag, Site Kit can place the new tag for you. Make sure you remove the old tag first.', 'google-site-kit' ), existingTag ) }</p>
```

Comment:
Capitalized "ID"

Reply:
I doubt that this translator comment is going to be picked up. It should be inside the brackets, preceding either `sprintf` or ideally the `__()` call.

```jsx
<p>{
sprintf(
	/* translators: %s: Analytics tag ID */
	__( 'An existing analytics tag was found on your site with the ID %s. If later on you decide to replace this tag, Site Kit can place the new tag for you. Make sure you remove the old tag first.', 'google-site-kit' ),
	existingTag
)
}</p>
```